readme.md

```diff
@@ -15,11 +15,17 @@ $ npm install --save dot-prop
 ```js
 var dotProp = require('dot-prop');
 
+// getter
 dotProp({foo: {bar: 'unicorn'}}, 'foo.bar');
 //=> 'unicorn'
 
-dotProp({foo: {bar: 'a'}}, 'foo.notDefined.deep');
```

Comment:
I think, if you want an error, you could just type `f1.foo.baz.bar` - bam! error.